generator/types.go

```diff
@@ -342,13 +342,6 @@ func (t *typeResolver) resolveArray(schema *spec.Schema, isAnonymous, isRequired
 	result.IsNullable = false
 	result.IsEmptyOmitted = t.IsEmptyOmitted(schema)
 
-	if !isAnonymous {
```

Comment:
This section is rather sensitive.
Could you please explain why you needed to remove this?

Reply:
This was added in my previous PR and I'm not sure what it exactly does, so reverted it to original state for now, as it does not affect result of my case.

Reply:
Be extremely cautious here, because types maybe primitives with pointers. There are some flaws here and it is difficult to modify this without changing many other stuff.

Reply:
This was code that was added by @moskal91 in his previous PR and now removed.
There is no risk afaik with this